src/gui/transferlistmodel.cpp

```diff
@@ -461,6 +461,10 @@ QVariant TransferListModel::data(const QModelIndex &index, const int role) const
         if (index.column() == TR_NAME)
             return getIconByState(torrent->state());
         break;
+    case Qt::ToolTipRole:
+        if (index.column() == TR_NAME)
```

Comment:
what about the following? (not tested)
```c++
case Qt::DisplayRole:
case Qt::ToolTipRole:
    return displayValue(torrent, index.column());
```
This will enable tooltips for all columns.

Reply:
I don't think that will be very useful to show tooltips for each column.... especially with exactly the same info as 'DisplayRole', this will cause a mess with duplicated info.
let's keep tooltips only for names for now, anything else can be added very easily later